torch/testing/_internal/custom_op_db.py

```diff
@@ -15,7 +15,7 @@
 )
 from torch import Tensor
 from torch.types import Number
-from typing import Sequence, Tuple
+from typing import *  # noqa: F403
```

Comment:
nit: import explicitly instead?

Reply:
I prefer the `import *` so I don't need to keep on adding imports